.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
```

Comment:
ehhhh i'm not a huge fan of this one but i'm down to try it and see how it goes

Reply:
Yeah, let's see how this works out, I think this could be a nice place to try stuff for Britecharts.